### ISSUE TITLE
[feature]: Create a sample app using GoLang and S3

### ISSUE THREAD

**Issue by Swpn0neel** [keploy/keploy#2110]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

A microservice in GoLang that handles file uploads to Amazon S3 and communicates with other services using SOAP. It needs to come with proper documentation.

### Why should this be worked on?

There's currently no such sample app with us.

### Repository

samples-go

**Comment by jaiakash** [comment]
Hi @Swpn0neel, I have a decentish understanding of Go and a good understanding of AWS. I am interested in this sample project since it will help me learn more about go. Can you please assign this to me?

**Comment by Swpn0neel** [comment]
Great, go ahead @jaiakash!!

**Comment by yaten2302** [comment]
@Swpn0neel, sir, may I collaborate with @jaiakash on this issue?

**Comment by Swpn0neel** [comment]
I've already assigned you to the node.js sample app @yaten2302! So currently you can work on that 😊

**Comment by yaten2302** [comment]
@Swpn0neel sir, I've completed my part in the other issues which were previously assigned to me, so now, may I collaborate with @jaiakash on this issue?

**Comment by jaiakash** [comment]
Sure @yaten2302, i will add you as collab.

**Comment by yaten2302** [comment]
Thanks @jaiakash , can you share the link of your forked repo? I'll create PRs on that and then you can create the final PR.
BTW, till where have you made the project? We can divide the work?

**Comment by jaiakash** [comment]
Hi, @yaten2302, thanks for your enthu. For now, could you give me some time to do this? 

There is small changes required in diff PR which is similar to this, can you help me with that. https://github.com/keploy/samples-java/pull/76 

There are some requested changes, which I can't do now because I am working on a different PR for the deploy build. 

Can you check it and update the Readme as requested? I have added you as collab.

Thanks

**Comment by yaten2302** [comment]
Yeah sure, @jaiakash , we can definitely colab on the other issue, which you gave 👍 
Actually, I haven't worked in Java much, so please forgive me if it will take some time for me to understand the code 🙏

**Comment by jaiakash** [comment]
Sure @yaten2302, The thing is that you don't need much java knowledge. That I have already created, check the documentation to project and `keploy` testing.
You can ping me on slack too.

**Comment by nitishmalang** [comment]
@Swpn0neel  can I work on this issue i can do it very fast

**Comment by Girishbari** [comment]
@Swpn0neel, if this is open, pls assign it to me. Also, what's the expected period to raise the PR
